partner_contact_birthplace/README.rst

```diff
@@ -0,0 +1,76 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+==========================
+Partner contact birthplace
+==========================
+
+This module extends the functionality of Odoo to support setting a bithplace city and allows you to benefit of a clearer API and UI.
+
+
+Installation
+============
+
+To install this module, you need to:
+
+* Install the OCA repository 'partner_contact'
+* Update your modules list.
+* Search and install this module.
+
+Configuration
+=============
+
+No configuration is needed.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Edit or create a partner.
+* Ensure it is **not** a company.
+* Go to the *Personal Information* sheet.
+* Set the birthplace there.
+
+For further information, please visit:
+
+* https://www.odoo.com/forum/help-1
+* https://github.com/OCA/partner-contact/
+
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/134/10.0
```

Comment:
```suggestion
   :target: https://runbot.odoo-community.org/runbot/134/12.0
```
This should point to v12.
A good way for generating the README correctly is using https://github.com/OCA/maintainer-tools#readme-generator